pkg/account/deployer/deployer.go

```diff
@@ -69,17 +69,28 @@ type client interface {
 }
 
 type AccountDeployer struct {
-	accClient client
-	idMap     idMap
-	logger    loggers.Logger
+	accClient  client
+	idMap      idMap
+	logger     loggers.Logger
+	maxWorkers int
 }
 
-func NewAccountDeployer(client client) *AccountDeployer {
-	return &AccountDeployer{
+func WithMaxWorkers(size int) func(*AccountDeployer) {
```

Comment:
nit: the "maxWorkers" naming makes sense on the Dispatcher, which has a worker pool and documentation defining how it creates workers. But to me it makes no sense on the AccountDeployer. Nothing else in the interface of the deployer would tell me what its workers are or why I would define WithMaxWorkers. 

Maybe a "max concurrent deploys" wording similar to the "max concurrent requests" would be easier to understand and document in relation to the Deployer.